pyicloud/base.py

```diff
@@ -18,6 +18,7 @@
 )
 from pyicloud.services import (
     FindMyiPhoneServiceManager,
+    FindMyFriendsServiceManager,
```

Comment:
```suggestion
    FindMyFriendsService,
```

FindMyiPhone service will be renamed in ##227 :wink: